### DISCUSSION TITLE
Encountering GUI strange rendering issue

### DISCUSSION THREAD

**Discussion by mcchang** [jesseduffield/lazygit#1181] [Q&A]
Hi!

Just starting to use lazygit, and encountered a strange rendering issue where the UI seems to rendering the box edges with dashes in iterm2 on Mac.  It doesn't happen in the default Terminal.app.  Does anyone know off the top of their head why this might be happening?

![image](https://user-images.githubusercontent.com/66332/110025710-be86af80-7cfd-11eb-89c2-6990adbb3356.png)

**Comment by mjarkk** [comment]
What's the output of:
```sh
echo $TERM
```

If it's not `xterm-256color` you could try running lazygit with:
```sh
TERM=xterm-256color lazygit
```

**Reply by mcchang** [reply]
`$TERM` is set to `xterm-color`.  Unfortunately setting `TERM` to `xterm-256color` doesn't work for me either

**Comment by jesseduffield** [comment]
@mcchang I would see if the latest release fixes your issue: we've swapped out the rendering library for a new one so this problem might be resolved

**Reply by mcchang** [reply]
> @mcchang I would see if the latest release fixes your issue: we've swapped out the rendering library for a new one so this problem might be resolved

@jesseduffield 0.27.4 doesn't seem to help unfortunately
